.github/workflows/autoformat.yml

```diff
@@ -71,6 +67,4 @@ jobs:
 
       - run: vendor/bin/php-cs-fixer fix
 
-      - uses: stefanzweifel/git-auto-commit-action@v5
-        with:
-          commit_message: Apply php-cs-fixer changes
+      - uses: autofix-ci/action@v1
```

Comment:
Requested app installation with @vladar 

It works smoooth https://github.com/simPod/graphql-php/pull/4

Reply:
Looks pretty sleek now. Is the installation ready?